src/platform/silabs/wifi/icd/WifiSleepManager.h

```diff
@@ -85,9 +123,11 @@ class WifiSleepManager
     CHIP_ERROR VerifyAndTransitionToLowPowerMode();
 
     static WifiSleepManager mInstance;
-    bool isCommissioningInProgress = false;
 
+    bool isCommissioningInProgress         = false;
     uint8_t mHighPerformanceRequestCounter = 0;
+
+    ApplicationCallbacks * mCallbacks = nullptr;
```

Comment:
Is this really meant to be multiple callbacks? If so a list, pool or some other structure should be used unless there is already so iterative mechanism that I am unaware of.

Reply:
No the design is for only a single pointer. In my interpretation the 's' was for multiple function in the application callback class which doesn't make much sense now that i think of it. 

Removing the s